CHANGELOG-nightly.md

```diff
@@ -2,6 +2,7 @@
 
 Note: Breaking changes between versions are indicated by "💥".
 
+- 💥[Bugfix] No longer track the Tutor version number in resource labels (and label selectors, which breaks the update of Deployment resources), but instead do so in resource annotations.
```

Comment:
As a rule of thumb, in the changelog I prefer when we describe _what_ we are fixing instead of _how_. The former is of more interest to the reader. Thus, a better changelog would be something along the lines of "Fix upgrading Kubernetes deployments when upgrading from one minor tutor version to the next".
But I don't want to block your PR for a changelog line, so let's merge this :)

Reply:
I agree, but in this case your abbreviated version doesn't contain any explanation as to why this is a _breaking_ change. (Why should a fix to upgrades be a breaking change?)

Reply:
Indeed. The changelog should also explain that the deployments should be deleted prior to upgrading.